src/python/test/behave_tests/features/upload_agent.feature

```diff
@@ -0,0 +1,11 @@
+@upload_agent
+Feature: 2: Upload agent example
+
+  Scenario: 2000_upload file with upload agent
+    Given CLI login as user qa_admin_org1
+    And CLI create new project
+      | name      | --select |
+      | [default] |          |
+    And generate file big 2 Bytes named zzz.file
```

Comment:
why not `generate big 2 Bytes file named zzz.file`?

Reply:
It is an existing step, phrased like that...

Reply:
weird by ok :D I guess it's not worth to fix it now